python/cudf/cudf/utils/dtypes.py

```diff
@@ -29,6 +29,31 @@
     np.str_: pa.string(),
 }
 
+pa_pd_types_mapper = {
+    pa.uint8: pd.UInt8Dtype(),
+    pa.uint16: pd.UInt8Dtype(),
+    pa.uint32: pd.UInt8Dtype(),
+    pa.uint64: pd.UInt8Dtype(),
+    pa.int8: pd.UInt8Dtype(),
+    pa.int16: pd.UInt8Dtype(),
+    pa.int32: pd.UInt8Dtype(),
+    pa.int64: pd.UInt8Dtype(),
+    pa.bool_: pd.UInt8Dtype()
+}
```

Comment:
Should we add a `pa.string: pd.StringDtype()`?

Reply:
Removing this as we shouldn't actually need it for this PR.